app/_includes/prereqs/konnect-api-for-curl.md

```diff
@@ -1,11 +1,7 @@
 To use the copy, paste, and run the instructions in this how-to, you must export these additional environmental variables:
 
 ```sh
-export KONNECT_CONTROL_PLANE_URL=https://{region}.api.konghq.com
```

Comment:
I'd actually leave this one and then merge Tomek's PR here, which removes this variable from the regular konnect include: https://github.com/Kong/developer.konghq.com/pull/1210/files